Scripts/BuildPackages.ps1

```diff
@@ -0,0 +1,10 @@
+$repositoryRoot = Join-Path $PSScriptRoot ".."
+
+$projects = @("CSharpier.CLI", "CSharpier", "CSharpier.MSBuild")
```

Comment:
So, the reason you're doing this instead of `dotnet pack` on the solution is because `CSharpier.MsBuild` can't be in the `.sln`, right?

If so, a comment explaining that might help newcomers.